templates/users_configMap.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+data:
+  mapRoles: |
+    - rolearn: {{ eks_iam_role_arn_worker }}
+      username: system:node: {{'{{'}}EC2PrivateDNSName{{'}}'}}
+      groups:
+       - system:bootstrappers
+       - system:nodes
+  mapUsers: |
+{% if eks_iam_users is defined %}
+{% for user in eks_iam_users_arn %}
+    - userarn: arn:aws:iam::{{ eks_aws_account_id }}:user/{{ user }}
```

Comment:
well I believe it is good way to insert its aws account id here so the user/dev won't need to change the template.

do you all agree ? is there any better way ? I am open to suggestions.